server/webserver.go

```diff
@@ -1074,19 +1074,15 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 
 	mux.HandleFunc("/EthChainID", func(w http.ResponseWriter, r *http.Request) {
 		if s.LivepeerNode.Eth == nil {
-			w.Write([]byte("0"))
+			w.Write([]byte("offchain"))
```

Comment:
Since the other return values for this handler are chain ID integers and we already use [0 as the chain ID](https://github.com/livepeer/go-livepeer/blob/c4a3004f8318220b42eacea33d392049cb7fa188/cmd/livepeer/livepeer.go#L306) in off-chain mode, WDYT about still returning 0 here for consistent return types from the handler and changing [checkNet()](https://github.com/livepeer/go-livepeer/blob/dacb1cb745503e730f9b342c2ad6e0079a03b1ab/cmd/livepeer_cli/livepeer_cli.go#L176) to check for 0?

Reply:
fixed in 78d85ab